test.txt

```diff
@@ -0,0 +1,56 @@
+﻿============================= test session starts =============================
```

Comment:
@Ace314159 is this supposed to be here

Reply:
Oh whoops, no I had it for testing. That's what I get for using the command line haha